peachjam/resources.py

```diff
@@ -86,7 +87,7 @@ def clean(self, value, row=None, **kwargs):
                     elif mime in DOC_MIMETYPES:

```

Comment:
Does DOC_MIMETYPES need to be updated to include RTF?

Reply:
I've added it. I thought that we would have to create a separate pipline for .rtf but adding the mimetype now extracts the text correctly. :100: